trunner/harness/plo.py

```diff
@@ -132,6 +132,23 @@ def erase(self, device: str, offset: int, size: int, timeout: Optional[int] = No
             e.cmd = cmd
             raise e
 
+    # fmt: off
```

Comment:
I can't suggest it in the right place, please move it above `PloInterface` class. I've got this solution from @nalajcie and it seems to be much more suitable there.

```suggestion
@dataclass(frozen=True)
class PloJffs2CleanmarkerSpec:
    """Represents the JFFS2 cleanmarkers image specs to be used for formatting"""

    start_block: int
    number_of_blocks: int
    block_size: int
    cleanmarker_size: int

    def __str__(self):
        """Generate spec as plo jffs2 command argument"""
        return f"{self.start_block}:{self.number_of_blocks}:{self.block_size}:{self.cleanmarker_size}"
    # fmt: off
```

Reply:
__str__ not added?

Reply:
Instead of using defined str we can use join and map as elements of cmd like : `':'.join(map(str, cleanmarkers))`

Reply:
I see that you applied some other approach, but the one suggested above seems to be more readable.

Reply:
def __str__(self):
        """Generate spec as plo jffs2 command argument"""
        return ':'.join(map(str,self.__dict__.values())